invisible_cities/core/core_functions.py

```diff
@@ -77,6 +82,54 @@ def in_range(data, minval=-np.inf, maxval=np.inf, left_closed=True, right_closed
     return lower_bound & upper_bound
 
 
+def check_if_values_in_interval(data         : np.ndarray,
+                                minval       : float   ,
+                                maxval       : float   ,
+                                display_name : str = '',
+                                strictness   : Strictness = Strictness.stop_proccess,
+                                **kwargs)->bool:
+    """
+    Checks whether input values are all inside the interval (minval, maxval).
+
+    Parameters
+    ----------
+    data : np.array
+        Input array to check.
+    minval: float
+        Lower limit of the interval.
+    maxval: float
+        Upper limit of the interval.
+    display_name: string
+        Name for the message to print if exception raises.
+    strictness: Strictness
+        If 'warning', function returns a False if the criteria
+        is not match. If 'stop_proccess' it raises an exception.
+    Returns
+    ----------
+        True if values are in the interval. False if not and strictness
+        is set to 'warning'. Otherwise, it raises an exception.
+    """
+
+    values_in_interval = in_range(data, minval, maxval, **kwargs)
+    outsiders_mask     = np.logical_not(values_in_interval)
+    outsiders_list     = data[outsiders_mask]
+    n_outsiders        = np.sum(outsiders_mask)
```

Comment:
```suggestion
    outliers   = data[~values_in_interval]
    n_outliers = len(outliers)
```

and change names below.